test/cmdlineTests/exp_base_literal/output

```diff
@@ -210,83 +210,83 @@ object "C_81" {
-                let _11 := vloc_d_10
+                let _11 := var_d_10
                 let expr_49 := _11
                 let _12 := convert_t_rational_115792089237316195423570985008687907853269984665640564039457584007913129639935_by_1_to_t_uint256(expr_48)
                 let expr_50 := checked_exp_t_rational_115792089237316195423570985008687907853269984665640564039457584007913129639935_by_1_t_uint256(expr_49)
-                let vloc_z_42 := expr_50
+                let var_z_42 := expr_50
                 let expr_53 := 0x00
                 let expr_54 := expr_53
-                let _13 := vloc_a_4
+                let _13 := var_a_4
                 let expr_55 := _13
                 let _14 := convert_t_rational_0_by_1_to_t_uint256(expr_54)
                 let expr_56 := checked_exp_t_rational_0_by_1_t_uint256(expr_55)
-                vloc_w_22 := expr_56
+                var_w_22 := expr_56
                 let expr_57 := expr_56
                 let expr_60 := 0x01
                 let expr_61 := 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
                 let expr_62 := expr_61
-                let _15 := vloc_b_6
+                let _15 := var_b_6
                 let expr_63 := _15
                 let _16 := convert_t_rational_minus_1_by_1_to_t_int256(expr_62)
                 let expr_64 := checked_exp_t_rational_minus_1_by_1_t_uint256(expr_63)
-                vloc_x_28 := expr_64
+                var_x_28 := expr_64
                 let expr_65 := expr_64
                 let expr_68 := 0x01
-                let _17 := vloc_c_8
+                let _17 := var_c_8
                 let expr_69 := _17
                 let _18 := convert_t_rational_1_by_1_to_t_uint256(expr_68)
                 let expr_70 := checked_exp_t_rational_1_by_1_t_uint256(expr_69)
-                vloc_y_36 := expr_70
+                var_y_36 := expr_70
                 let expr_71 := expr_70
-                let _19 := vloc_w_22
+                let _19 := var_w_22
                 let expr_73 := _19
                 let expr_77_component_1 := expr_73
-                let _20 := vloc_x_28
+                let _20 := var_x_28
                 let expr_74 := _20
                 let expr_77_component_2 := expr_74
-                let _21 := vloc_y_36
+                let _21 := var_y_36
                 let expr_75 := _21
                 let expr_77_component_3 := expr_75
-                let _22 := vloc_z_42
+                let _22 := var_z_42
                 let expr_76 := _22
                 let expr_77_component_4 := expr_76
-                vloc__13 := expr_77_component_1
-                vloc__15 := expr_77_component_2
-                vloc__17 := expr_77_component_3
-                vloc__19 := expr_77_component_4
+                var__13 := expr_77_component_1
+                var__15 := expr_77_component_2
+                var__17 := expr_77_component_3
+                var__19 := expr_77_component_4
```

Comment:
I noticed that the diff in this test is only related to the diff in `ir/Common.cpp.` I suspect the name simplifier step is not even called.

Reply:
Ah I guess this is because the args for this test is `--ir` i.e., there is no `--ir-optimized` which iiuc is necessary to invoke the name simplifier.